### DISCUSSION TITLE
how to limit search result between MM relation fields.

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#17269] [How to]
<img width="418" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/18006681/97e7cd3e-8937-4440-aab2-baeb2a891c8f">

here is the description of my problem:
account:contact  is 1m relationship. when I click contact,  it will show up all the result limited by account, works fine.  
account: sales order is mm relationship. when I click sales order, it just shows me all the sales order result,  how to limit it by account?

please help.

**Comment by deepti4666** [comment]
I also want to do this, but on the good dispatched note.

**Comment by newminder** [comment]
did you solve that problem
